deployment/README.md

```diff
@@ -8,149 +8,349 @@ proxy](https://doc.traefik.io/traefik/) as a reverse proxy, which listens
 on port 80 and 443.  Based on some labels, traefik routes the domains to the
 corresponding docker containers.
 
+## Requirements
+
+This guide requires a Linux-system, where `docker` and `docker-compose` are installed.
+Ensure, that your system is up to date.
+
+> TODO
```

Comment:
Please do not try to document docker installation, it is soooo different for each system.

Please also use `openssl rand -base64 32` or `-hex 16` or so, instead of `pwgen` as that tends to be available everywhere ;)